website/client/components/chat/copyAsTodoModal.vue

```diff
@@ -50,18 +55,22 @@ export default {
     },
   },
   methods: {
+    ...mapActions({
+      createTask: 'tasks:create',
+    }),
     close () {
       this.$root.$emit('hide::modal', 'copyAsTodo');
     },
     saveTodo () {
-      // let newTask = {
-      //   text: this.text,
-      //   type: 'todo',
-      //   notes: this.notes,
-      // };
+      const newTask = taskDefaults({type: 'todo', text: this.text, notes: this.notes});
 
-      //  @TODO: Add after tasks: User.addTask({body:newTask});
-      // @TODO: Notification.text(window.env.t('messageAddedAsToDo'));
+      this.createTask(newTask);
+      this.$store.dispatch('snackbars:add', {
+        title: 'Copy Task',
+        text: this.$t('messageAddedAsToDo'),
+        type: 'success',
+        timeout: true,
+      });
```

Comment:
sorry for the delay reviewing this, the code looks good but can you make sure the `text` and `notes` in the modal get reset after the task is created? Thanks